test/puppetlabs/puppetdb/command_test.clj

```diff
@@ -813,6 +822,21 @@
           (let [result (query-to-vec "SELECT certname,environment_id FROM factsets")]
             (is (= result [(with-env {:certname certname})])))))))
 
+  (deftest replace-facts-nonstandard-certname
```

Comment:
Not sure, but thought it might make sense to consider using "unusual" rather than "nonstandard" just to match our usage elsewhere (in the recent commits fixing other issues related to this feature, etc.).